code/graphics/gropengl.cpp

```diff
@@ -1463,42 +1438,31 @@ static void init_extensions() {
 	GLSL_version = ver;
 
 	// we require a minimum GLSL version
-	if (!is_minimum_GLSL_version()) {
+	if (GLSL_version < MIN_REQUIRED_GLSL_VERSION) {
```

Comment:
That would be the only location where that function could be used so I'll keep the current version.